### PR TITLE
feat(spark): added spark chart to helm charts

### PR DESCRIPTION
**Pull request by chaitanyaenr**
Spark chart for Kubernetes.
Tested on kubernetes cluster( AWS ), also tested using helm.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
Need for zeppelin service and controller: Apache Zeppelin is a new and incubating multi-purposed web-based notebook which brings data ingestion, data exploration, visualization, sharing and collaboration features to Spark. I added the home page in Chart.yaml.

File: spark/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: spark
+home: http://spark.apache.org/, https://zeppelin.incubator.apache.org/
```
Comment (bacongobbler) [inline spark/Chart.yaml]: same comment as https://github.com/helm/charts/pull/74#discussion_r53874292; this cannot be an array. We must choose one.

**Comment by bacongobbler** [comment]
You might want to put that in the chart README so people installing this chart understand why zeppelin is necessary.

**Comment by chaitanyaenr** [comment]
Yeah, sounds good. Made the changes.

**Comment by sgoings** [comment]
Looks good! Tested manually. @bacongobbler can you give this another once over?

**Pull request merged**